app/matplotlib.ipynb

```diff
@@ -339,18 +348,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
-   "outputs": [],
    "source": [
     "# write your code here\n"
```

Comment:
Task 9 is incomplete. You need to implement the scatter plot matrix using the provided data file. Consider using `pandas.plotting.scatter_matrix` or `seaborn.pairplot` for this task.